### PR TITLE
Refactor css units

### PR DESCRIPTION
**Pull request by bazelajoanna**
Changes in css units based on following rules:

- REM - is a main unit in our project;

- EM - we use it just in media-queries;

- PX - we use it only  to declaring very precise things like border-radius;

- % - we use it to declaring font-size in global styles, sometimes in width/height and in style calculations.

### CODE REVIEW

File: src/assets/styles/breakpoints.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
   base: 0,
-  mobile: "320px",
-  desktopTablet: "600px",
-  desktopL: "900px",
-  desktopXL: "1280px",
+  mobile: "20em",
+  desktopTablet: "37.5em",
+  desktopL: "56.25em",
+  desktopXL: "80em",
 };
```
Comment (evulins) [inline src/assets/styles/breakpoints.js:7]: 👍

File: src/assets/styles/global-styles.js [hunk 2]
```diff
@@ -54,7 +61,7 @@ const GlobalStyle = createGlobalStyle`
     margin: 0;
     padding: 0;
     border: 0;
-    font-size: 100%;
+    /* font-size: 62.5%; */
```
Comment (evulins) [inline src/assets/styles/global-styles.js]: I think we can delete this line.

File: src/assets/styles/global-styles.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ const GlobalStyle = createGlobalStyle`
     min-height: 100vh;
     width: 100%;
     color: #1F1F1F;
+
     /* -webkit-font-smoothing: antialiased; */
```
Comment (evulins) [inline src/assets/styles/global-styles.js]: Probably we don't need it right now

**Review by evulins — COMMENTED** [review]
Great job :) 👏  I have just small comments :)

File: src/components/Footer/styledComponents.tsx [hunk 4]
```diff
@@ -10,47 +10,47 @@ const FooterWrapper = styled.div`
...
 
 const ContactWrapper = styled.div`
-  padding-left: 2em;
-  margin-top: 1em;
+  padding-left: 3.2rem;
+  margin-top: 1.6rem;
   @media ${minDevice.desktopTablet} {
     margin-top: 0;
-    padding-right: calc(83px - 1.5em);
+    padding-right: calc(8.3rem - 2.4rem);
```
Comment (franiakat) [inline src/components/Footer/styledComponents.tsx]: Can we calculate that on our own? 🥦 😁

File: src/components/SearchAutocomplete/styledComponents.tsx [hunk 5]
```diff
@@ -14,14 +14,14 @@ import { ReactComponent as ClearIcon } from "../../assets/img/icons/remove-icon.
...
+  height: 8rem;
   box-shadow: 0px 4px 15px 0px rgba(22, 28, 86, 0.2);
   z-index: 10;
   @media ${minDevice.desktopTablet} {
-    width: 22.5em;
+    width: 36rem;
   }
   @media ${minDevice.desktopL} {
-    width: 31.4375em;
+    width: 50.3rem;
```
Comment (franiakat) [inline src/components/SearchAutocomplete/styledComponents.tsx]: 😍

**Review by franiakat — COMMENTED** [review]
🦖 nice, I just had one doubt 🙃

**Comment by franiakat** [comment]
Just the suggestion list items are very small 🤔

File: src/components/Footer/styledComponents.tsx (hunk 4 above)
Comment (bazelajoanna) [reply src/components/Footer/styledComponents.tsx]: Yes, we can :D

**Comment by bazelajoanna** [comment]
> Just the suggestion list items are very small 🤔

I've fixed it in commit "Change breakpoints and span font size". Hmm 🤔

**Review by evulins — APPROVED** [review]
👏

**Review by franiakat — APPROVED** [review]
sqash

**Pull request merged**